hledger-lib/Hledger/Read/Common.hs

```diff
@@ -220,28 +227,47 @@ definputopts = InputOpts
     , new_               = False
     , new_save_          = True
     , pivot_             = ""
+    , forecast_          = Nothing
     , auto_              = False
     , balancingopts_     = def
     , strict_            = False
     }
 
-rawOptsToInputOpts :: RawOpts -> InputOpts
-rawOptsToInputOpts rawopts = InputOpts{
-     -- files_             = listofstringopt "file" rawopts
-     mformat_           = Nothing
-    ,mrules_file_       = maybestringopt "rules-file" rawopts
-    ,aliases_           = listofstringopt "alias" rawopts
-    ,anon_              = boolopt "anon" rawopts
-    ,new_               = boolopt "new" rawopts
-    ,new_save_          = True
-    ,pivot_             = stringopt "pivot" rawopts
-    ,auto_              = boolopt "auto" rawopts
-    ,balancingopts_     = def{ ignore_assertions_ = boolopt "ignore-assertions" rawopts
-                             , infer_prices_      = not noinferprice
-                             }
-    ,strict_            = boolopt "strict" rawopts
+rawOptsToInputOpts :: Day -> RawOpts -> InputOpts
```

Comment:
I guess this was done for a reason - is it used somewhere ?

Reply:
Sorry, can you clarify what you're referring to? Do you mean the addition of the `Day` argument? it is used in parsing any smart dates like `yesterday` in the forecast period.

Reply:
Yes I mean the addition of the Day argument. (I was trying to reference that commit/just the change of type signature, unsuccessfully). I just couldn't see where it was used but I guessed it was.